index.html

```diff
@@ -2406,6 +2634,1425 @@ <h2>
+            </h5>
+            <pre class="cddl">
+              emulation.SimulateDevice = {
+                method: "emulation.simulateDevice",
+                params: emulation.SimulateGamepadParams,
+              }
+              Returns: deviceId: text
+            </pre>
+            <p>
+              The [=remote end steps=] with command parameters |params| are:
+            </p>
+            <ol>
+              <li>
+                Let |source| be the result of [=create a gamepad input source=] with |params|.
+              </li>
+              <li>
+                Let |simulatedGamepads| be the [=simulated gamepad map=].
+              </li>
+              <li data-cite="webcrypto">
+                Let |deviceId:string| be the result of calling [=generate a random UUID=].
+              </li>
+              <li>
+                [=map/Set=] |simulatedGamepads|[|deviceId|] to |source|.
+              </li>
+              <li>
+                Run the steps [=when a gamepad becomes available on the system=] for |gamepad|.
+              </li>
+              <li>
+                Return |deviceId|.
+              </li>
+            </ol>
+          </section>
+          <section>
+            <h5>
+              The emulation.removeSimulatedDevice Command
+            </h5>
+            <pre class="cddl">
+              emulation.RemoveSimulatedDevice = {
+                method: "emulation.removeSimulatedDevice",
+                params: emulation.RemoveSimulatedDeviceParams,
+              }
+              Returns: void
+            </pre>
+            <p>
+              The [=remote end steps=] with command parameters |params| are:
+            </p>
+            <ol>
+              <li>
+                Let |simulatedGamepads| be the [=simulated gamepad map=].
+              </li>
+              <li>
+                If |simulatedGamepads|[|params|.|deviceId|] does not [=map/exist=], abort these steps.
+              </li>
+              <li>
+                Let |gamepad| be |simulatedGamepads|[|params|.|deviceId|].
+              </li>
+              <li>
+                Run the steps [=when a gamepad becomes unavailable on the system=] with |gamepad|.
+              </li>
+              <li>
+                [=map/Remove=] |simulatedGamepads|[|params|.|deviceId|].
+              </li>
+            </ol>
+          </section>
+        </section>
+        <section data-cite="infra">
+          <h4>
+            Events
+          </h4>
+          <section>
+            <h5>
+              The emulation.simulatedDeviceEvent Event
+            </h5>
+            <pre class="cddl">
+              emulation.SimulatedDeviceEvent = {
+                method: "emulation.removeSimulatedDevice",
+                params: emulation.SimulatedDeviceEventParams,
+              }
+            </pre>
+            <p>
+              To get <dfn>the identifier for a simulated gamepad</dfn> |gamepad|:
```

Comment:
Optionally, we can also extend the definition of "simulated gamepad" as "Simulated gamepad has an associated [=UUID=] <dfn>simulated gamepad identifier</dfn>".